storagedriver/azure/blockid.go

```diff
@@ -0,0 +1,56 @@
+package azure
+
+import (
+	"encoding/base64"
+	"fmt"
+	"math/rand"
+	"sync"
+
+	azure "github.com/MSOpenTech/azure-sdk-for-go/clients/storage"
+)
+
+type blockIDGenerator struct {
+	pool map[string]bool
+	m    sync.Mutex
+}
+
+// Generate returns an unused random block id and adds the generated ID
```

Comment:
@stevvooe yeah, pool just helps retrying forever. however on the average case, the for loop in the `Generate` method below won't be executed more than twice because we are making use of globalRand (although we don't seed it) and since the blobs we are operating are changing all the time, generated ID is most likely going to be unique within the blob.

You're aware that there's a for loop in generate method that tries forever to not to generate a used ID, right? Even though the generated IDs collide, we'd keep retrying.

Do you think should I seed the globalRand?